tests/cypress/e2e/all/header.cy.ts

```diff
@@ -160,6 +163,50 @@ describe('Header', () => {
     })
   })
 
+  describe('Phishing Warning Banner', () => {
+    let route: AppRoute
+
+    beforeEach(() => {
+      const [width, height] = oneViewport()
+      viewport = [width, height]
+      cy.viewport(...viewport)
+      route = oneAppRoute()
+      cy.visitWithoutTestConnector(route)
+      waitIsLoaded(route)
+    })
+
+    it('should display the banner and allow dismissal', () => {
+      cy.get("[data-testid='phishing-warning-banner']", LOAD_TIMEOUT).should('be.visible')
+      // Click the banner to dismiss it
+      cy.get("[data-testid='phishing-warning-banner']").find('button').first().click()
+      cy.get("[data-testid='phishing-warning-banner']").should('not.exist')
+    })
+
+    it('should reappear after one month', () => {
+      // Set dismissal date to 31 days ago (more than one month)
+      const oneMonthAgo = Date.now() - 31 * 24 * 60 * 60 * 1000
+      dismissPhishingWarningBanner(oneMonthAgo)
+      cy.reload()
```

Comment:
ideally we can move the cy.visit to the tests so a reload is not needed. cy.visit can receive an object with `{onBeforeLoad: window => {...}}`